message-channel/index.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: pattern
+title: Message Channel
+folder: message-channel
+permalink: /patterns/message-channel/
+categories: Integration
+tags: Java
+---
+
+**Intent:** When two applications communicate using a messaging system they do it by using logical addresses
+of the system, so called Message Channels.
+
+![alt text](./etc/message-channel.png "Message Channel")
```

Comment:
I think the implementation is the same across all the patterns and we have no agreed convention on this.